lib/common/client/base.rb

```diff
@@ -55,11 +55,15 @@ def request(method, path, params = {}, headers = {})
         )
         raise Common::Exceptions::GatewayTimeout
       rescue Faraday::ClientError => e
-        client_error = Common::Client::Errors::ClientError.new(
-          e.message,
-          e.response&.dig(:status),
-          e.response&.dig(:body)
-        )
+        error_class = case e
```

Comment:
@rlbaker 👍 - we have some logic in EVSS services that can go away now that this is in. It won't break anything as the `Common::Client::Errors::ClientError` logic we have will catch it instead.